config/locales/provider_relationship_permissions.yml

```
@@ -2,4 +2,4 @@ en:
   provider_relationship_permissions:
     make_decisions: This allows an organisation to make offers, amend offers and rejection applications
     view_safeguarding_information: "This allows an organisation to see information disclosed in the ‘criminal convictions and professional misconduct’ section of the application"
-    view_diversity_information: "This allows an organisation to see information disclosed in the ‘equality and diversity’ section of the application"
+    view_diversity_information: "This allows an organisation to see information disclosed in the ‘equality and diversity’ questionnaire. This relates to the candidate's sex, ethnicity and disability status"
```

```suggestion
    view_diversity_information: "This allows an organisation to see information disclosed in the ‘equality and diversity’ questionnaire. This relates to the candidate’s sex, ethnicity and disability status"
```

A bit of a nitpick. The designers always point it out to me in my PRs 😂

---

Thanks for pointing that out, I would have never noticed!